_replication-plugin/permissions.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Permissions
+nav_order: 30
+---
+
+# Cross-cluster replication permissions
+
+You can use the [security plugin]({{site.url}}{{site.baseurl}}/security-plugin/index/) with cross-cluster replication to limit users to certain actions. For example, you might want certain users to only perform replication activity on the leader or follower cluster.
+
+Because cross-cluster replication involves multiple clusters, it's possible that clusters might have different security configurations. The following configurations are supported:
+
+- Security plugin fully enabled on both clusters
+- Security plugin enabled only for TLS on both clusters (`plugins.security.ssl_only`)
+- Security plugin absent or disabled on both clusters (not recommended)
+
+You can enable node-to-node encryption on both the leader and the follower cluster to ensure that replication traffic between the clusters is encrypted.
+
+## Basic permissions
+
+In order for non-admin users to perform replication activities, they need to be mapped to the appropriate permissions.  
+
+The security plugin has two built-in roles that cover most replication use cases: `cross_cluster_replication_leader_full_access` which provides replication permissions on the leader cluster, and `cross_cluster_replication_follower_full_access` which provides replication permissions on the follower cluster. For descriptions of each, see [Predefined roles]({{site.url}}{{site.baseurl}}/security-plugin/access-control/users-roles#predefined-roles).
+
+If you don't want to use the default roles, you can combine individual replication [permissions]({{site.url}}{{site.baseurl}}/replication-plugin/permissions/#replication-permissions) to meet your needs. Most permissions correspond to specific REST API operations. For example, the `indices:admin/plugins/replication/index/pause` permission lets you pause replication.
+
+## Map the leader and follower cluster roles
+
+The [start replication]({{site.url}}{{site.baseurl}}/replication-plugin/api/#start-replication) and [create replication rule]({{site.url}}{{site.baseurl}}/replication-plugin/api/#start-replication) are special cases because they involve background processes that require permissions on both the leader and the follower cluster. Therefore, when you perform one of these actions, you need to explicitly pass the  `leader_cluster_role` and
```

Comment:
I'd split this up a bit. Something like:

Start replication and create replication rule are special cases. They involve background processes that require permissions on both the leader and the follower cluster. When you perform one of these actions, you must explicitly pass the `leader_cluster_role` and `follower_cluster_role` in the request, which OpenSearch then uses in all backend replication tasks.